caom2/caom2/plane.py

```diff
@@ -1149,15 +1149,24 @@ def bandpass_name(self, value):
         self._bandpass_name = value
 
     @property
-    @deprecated(version='CAOM2.4', reason='Replaced by energy_bands')
+    @deprecated(version='CAOM2.4',
+                reason='Replaced by energy_bands, gone in 2.5')
     def em_band(self):
         """ EM Band """
-        return None
+        result = None
+        if len(self._energy_bands) == 1:
+            result = next(iter(self._energy_bands)).value
+        return result
 
     @em_band.setter
-    @deprecated(version='CAOM2.4', reason='Replaced by energy_bands')
+    @deprecated(version='CAOM2.4',
+                reason='Replaced by energy_bands, gone in 2.5')
     def em_band(self, value):
-        pass
+        if len(self._energy_bands) == 0:
+            if value is not None:
+                self._energy_bands.add(value)
+        else:
```

Comment:
when len==1, I think we can just do a replace.

Reply:
Returned the implementation to 'do nothing', as that seems to be the only way to not interfere with the checksums.